circle.yml

```diff
@@ -0,0 +1,27 @@
+machine:
+  services:
+    - docker
+
+deployment:
+  production:
```

Comment:
probably wise not to deploy on prod atm since it's an older version of docker which doesn't do log-options. That's why I have a different template for witan-app prod.